compiler/rustc_const_eval/src/check_consts/check.rs

```diff
@@ -663,8 +657,9 @@ impl<'tcx> Visitor<'tcx> for Checker<'_, 'tcx> {
                     // typeck ensures the conditions for calling a const trait method are met,
                     // so we only error if the trait isn't const. We try to resolve the trait
                     // into the concrete method, and uses that for const stability checks.
-                    // FIXME(effects) we might consider moving const stability checks to typeck as well.
-                    if tcx.features().effects() && trait_is_const {
+                    // FIXME(const_trait_impl) we might consider moving const stability checks
+                    // to typeck as well.
+                    if tcx.features().const_trait_impl() && trait_is_const {
```

Comment:
The `effects` feature name still appears a few lines down in this file

https://github.com/rust-lang/rust/blob/6b96103bf3b0746e443f9bb86b532be2c536113a/compiler/rustc_const_eval/src/check_consts/check.rs#L680-L684